probe-rs/src/flash/mod.rs

```diff
@@ -4,12 +4,14 @@
 
 pub mod builder;
 pub mod download;
+pub mod error;
 pub mod flasher;
 pub mod loader;
 pub mod progress;
 
 pub use builder::*;
```

Comment:
Please remove the imports which are not needed anymore.

Reply:
Which ones do you mean? The ones annotated as unused? Already done in the next PR, I messed up.